lib/anyway/config.rb

```diff
@@ -43,22 +43,22 @@ def config_name(val = nil)
       #
       #   my_config = Anyway::Config.for(:my_app)
       #   # will load data from config/my_app.yml, secrets.my_app, ENV["MY_APP_*"]
-      def for(name)
-        new(name, false).load_from_sources
+      def for(explicit_values: {}, name: nil)
```

Comment:
I don't think we need explicit values here, 'cause this method returns a plain Hash

Let's leave as is.